fixit/cli/add_new_rule.py

```diff
@@ -64,10 +65,19 @@ def is_path_exists(path: str) -> Path:
         return filepath
 
 
-def create_rule_file(file_path: Path) -> None:
+def is_valid_name(name: str) -> str:
+    """Check for valid rule name """
+    if name.endswith(("Rule", "Rules", "rule", "rules")):
+        raise NameError("Please enter rule name without the suffix, `rule` or `Rule`")
```

Comment:
It would be more consistent if you use single quotes instead of backticks (just like KeyError etc.)

Reply:
Also, NameError doesn't suit well, maybe use ValueError?